subsystem/host_test.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/openshift/assisted-service/models"
 )
 
-var _ = Describe("Host tests", func() {
+var _ = FDescribe("Host tests", func() {
```

Comment:
why?

Reply:
Well it's something we can consider using, but should standardize it

Reply:
Corrected